sgkit/stats/ld.py

```diff
@@ -0,0 +1,59 @@
+import math
+
+import numba
+import numpy as np
+from numba import njit
+
+from sgkit.typing import ArrayLike
+
+
+@njit(nogil=True, fastmath=True, cache=True, locals={"m0": numba.float32, "m1": numba.float32, "v0": numba.float32, "v1": numba.float32, "cov": numba.float32, "n": numba.int32})  # type: ignore
```

Comment:
Does it make much difference to declare these as float32s? I'd have thought it would be safer to do the calc in double precision, and I'd be surprised if it made any perf difference. Particularly for accumulating values in a CPU register, I can't see any point in doing this in single precision.

Reply:
Ah I forgot to mention here that I needed to declare the local variables as float32 to get concordance with scikit-allel's `locate_unlinked` function. I agree it shouldn't make much of a performance difference though. See #532.